data/prrajput1199.json

```
@@ -0,0 +1,25 @@
+{
+  "name": "Pratik Rajput",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Open-Source enthusiastic",
+  "avatar": "https://github.com/prrajput1199.png"
+  "links":[
+  {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/PratikR61035031",
+      "icon": "FaTwitter"
+  }
```

Please add `,` after the `}`
```suggestion
  },
```